Cartfile

```diff
@@ -16,7 +16,10 @@ github "Leanplum/Leanplum-iOS-SDK"          ~> 2.7.2
 github "mozilla-services/shavar-prod-lists" "72.0"
 # Use release version of application-services
 github "mozilla/application-services"      "v61.0.7"
-github "mozilla/glean"                     "v31.3.0"
+# github "travis79/glean"                     "Bug1637539"
```

Comment:
I want to build it myself, is this the branch I use?
I want to try a local archive build with and without Glean static and see if I see a difference in binary size. If symbol stripping is working properly during linkage I would expect it to shrink, if not that sounds like there is a problem. Even if there is a symbol stripping problem, just removing another dylib (which dyld has to map and relocate all the symbols, etc) will be the startup benefit.

Reply:
I am on iOS14 branch right now, so I want to try it on that branch which is why I would not use the pre-built glean lib

Reply:
Yes, that's the branch.  I was just using the binary for testing out BuddyBuild to ensure it would build correctly.

Reply:
For reference, that branch is the base of the diff for this PR updating Glean: https://github.com/mozilla/glean/pull/1107